### ISSUE TITLE
feedback, rough draft

### ISSUE THREAD

**Issue by hardin47** [latlio/SurvivalAnalysis#17]
Hi, @MadHobbs @latlio ,

Great job on the project so far!!!  It is really a pleasure to read.  You two seem to have learned a ton, and you've really gone above and beyond with respect to additional analyses on the dataset.  

I do have a few questions about some of those extended analyses and a few other clarifications.  Here goes ...  :)

* When you give a test and p-value (e.g., middle of page 9 of pdf w chi-sq is 91.05), be specific about what you are testing (that is, give the null hypothesis or conclusion or whatever).  Right after that, you do another chi-sq, but this time it seems like it is a nested test, where above it seems to be a full-model test.  Anyway, be specific.

* Not absolutely necessary, but do you have a good way to explain what is being measured on the "residual" axis?  Something minus time?  and is there any sense for what a "big" residual is?  Or for that matter, a "big" dfbeta?

* Bias in parameters (e.g., sample max vs true max) is different from bias in overfitting.  I'm not sure that bootstrapping can estimate the bias in overfitting.  You can estimate bias in overfitting from cross validation... but bootstrapping still has all the same data, so I'm not sure what it can tell you about new data.

* Can you be more specific about the definition of D_xy?  How is the probability of concordance calculated?  Also, how is "optimism" technically calculated?

* Figure 8:  I think I know what you did because of the conversation we had.  But you need to explain how you have hundreds of SEs by which you can make boxplots.  And also, can you contextualize why the boxplots are so much closer to the blue dot than the red dot? It makes sense to me, and I don't find it worrisome, I just think it is worth mentioning because the plot makes it seem like the red is off.

* Figure 9:  I love this plot  But can you add a caption?  Also, my guess is that you did +/- 2*SE for the CIs?  Say that because there are many different ways to create CIs, and it isn't clear what you did.

* Your comment on Fig 9:  you actually have NO idea whether the beta estimates ("b") are in the center or the tail of the TRUE sampling distribution.  What the plots tell you is that using normal theory seems like a reasonable thing to do.  If this particular comment isn't clear, I'll draw you a picture.

* Figure 10 ...  First of all, I'm a little bit confused about the layout of your report.  Why are we back to the model building?  Maybe just because the bootstrap validated it?  Also, what is being plot?  Seems like in (a) log(HR) is on the y-axis and in (b) and (c) log(HR) is on the x-axis?  And then maybe there are some SE bars?  Can you describe how those plots were created (and what is the ribbon for cd4? how did you get SE of the log(HR) as a function of cd4?)

* Figure 11...  I'm not even totally clear how one calculates the predicted survival.  Don't you need to know/estimate h(t) in order to get S(t)?  Or is there some way to estimate the *change* in survival without knowing h(t)?

* XGBoost:  interestingly, one comment here is related to both the main analysis and Lathan's bootstrapping:  **what is a residual?**  I don't actually think that you use residuals in the end, but you do discuss the idea of a residual when you introduce XGBoost, and residuals are a pretty tough concept in CoxPH.  Maybe reflecting on why/how residuals are tough to think about will help you understand why they had to do the other complicated stuff.

* You use F to mean two different things.  Er, maybe you do??  I'd like to walk through your work together.  Any chance we can do that?  It's hard for me to give feedback in the GitHub text box ...  

* There are a few places where a good read would help the language part.  I think you say "my" instead of "may".  And some places you say "I" other places you say "we".  Spell check.  Etc.

Nice job, team!  Thanks for the really great work and for sharing it with me.
